frontend/lib/src/components/shared/StreamlitMarkdown/Heading.test.tsx

```diff
@@ -33,7 +33,7 @@ const getHeadingProps = (
     anchor: "some-anchor",
     tag: "h1",
     body: `hello world
-             this is a new line`,
+this is a new line`,
```

Comment:
The whitespace here was triggering a code block. The lazy loading of code dependencies (triggering a skeleton) apparently caused some flakiness.